packages/ui/useScroll/useScroll.js

```diff
@@ -1076,11 +1076,14 @@ const useScrollBase = (props) => {
 
 		if (scrollMode === 'translate') {
 			showThumb(bounds);
+			if (scrollContentHandle.current && scrollContentHandle.current.setScrollPositionTarget) {
+				scrollContentHandle.current.setScrollPositionTarget(targetX, targetY);
+			}
 
 			if (animate) {
 				mutableRef.current.animator.animate(scrollAnimation(mutableRef.current.animationInfo));
 			} else {
-				scroll(targetX, targetY, targetX, targetY);
+				scroll(targetX, targetY);
```

Comment:
Now  `scroll` function get two parameter always.
I think the third parameter make confusing to understand further behavior. 
`scroll = (left, top, ...rest) => {`
How about remove the `...rest` also?

Reply:
For `translate` VirtualList, the `rest` should be passed to the `setScrollPosition`.

Reply:
So the `this.scroll` usage is
```
this.scroll(targetX, targetY, targetX, targetY);
```

Reply:
? I think @jeonghee27 is right.

Reply:
Now we are setting `scrollPositionTarget` seperately.

Reply:
Oh, my misunderstanding. I fixed it.